### PR TITLE
Replace more vendor naming

### PR DESCRIPTION
**Pull request by nickvergessen**
Reported at https://www.transifex.com/nextcloud/nextcloud/translate/#de/core/90994078

@MorrisJobke @jospoortvliet

### CODE REVIEW

**Comment by rullzer** [comment]
:+1:

**Comment by LukasReschke** [comment]
LGTM

File: core/js/tests/specs/setupchecksSpec.js [hunk 1]
```diff
@@ -309,7 +309,7 @@ describe('OC.SetupChecks tests', function() {
 
 			async.done(function( data, s, x ){
 				expect(data).toEqual([{
-					msg: 'The reverse proxy headers configuration is incorrect, or you are accessing ownCloud from a trusted proxy. If you are not accessing ownCloud from a trusted proxy, this is a security issue and can allow an attacker to spoof their IP address as visible to ownCloud. Further information can be found in our <a target="_blank" rel="noreferrer" href="https://docs.owncloud.org/foo/bar.html">documentation</a>.',
+					msg: 'The reverse proxy headers configuration is incorrect, or you are accessing Nextcloud from a trusted proxy. If you are not accessing Nextcloud from a trusted proxy, this is a security issue and can allow an attacker to spoof their IP address as visible to Nextcloud. Further information can be found in our <a target="_blank" rel="noreferrer" href="https://docs.owncloud.org/foo/bar.html">documentation</a>.',
```
Comment (MariusBluem) [inline core/js/tests/specs/setupchecksSpec.js]: Shouldnt the documentation link to Nextcloud 🤔

Comment (nickvergessen) [inline core/js/tests/specs/setupchecksSpec.js]: This is only in tests

**Pull request merged**